src/system/utils/handlebars/index.ts

```diff
@@ -490,6 +501,7 @@ export async function preloadHandlebarsTemplates() {
         'systems/cosmere-rpg/templates/item/goal/partials/goal-details-tab.hbs',
         'systems/cosmere-rpg/templates/item/power/partials/power-details-tab.hbs',
         'systems/cosmere-rpg/templates/item/path/partials/path-details-tab.hbs',
+        'systems/cosmere-rpg/templates/item/talent-tree/partials/talent-tree-node-tooltip.hbs',
         'systems/cosmere-rpg/templates/combat/combatant.hbs',
         'systems/cosmere-rpg/templates/chat/parts/roll-details.hbs',
         'systems/cosmere-rpg/templates/chat/parts/chat-card-header.hbs',
```

Comment:
Note that if we merge #151 and this PR around the same time, we need to move this to the new format for template registration, which is putting the path into the TEMPLATES object so the registration helper can take it automatically from there

Reply:
Yeah let's merge chat cards in first, then I'll fix the merge issues on this PR